spec/gilded_rose_spec.js

```diff
@@ -1,7 +1,180 @@
 describe("Gilded Rose", function() {
 
-  it("should do something", function() {
-    update_quality();
-  });
+  describe('normal items (dexterity and Elixir)', () => {
+    it("+5 Dexterity Vest should update quality minus 1 and sell_in minus 1 with sell_In still above zero", function() {
+      items = []
+      items.push(new Item('+5 Dexterity Vest', 10, 20));
 
-});
+      update_quality();
+      expect(items[0]).toEqual(jasmine.objectContaining({
+        name: '+5 Dexterity Vest',
+        sell_in: 9,
+        quality: 19
+      }))
+    });
+    it('Exlixir should update quality minus 1 and sell_in minus 1 with sell_In still above zero', () => {
+      items = []
+      items.push(new Item('Elixir of the Mongoose', 5, 7));
```

Comment:
I kind of like this two-step approach.
I typically would do this:

```JavaScript
items = [new Item('Elixir of the Mongoose', 5, 7)]
```

But it's easy to miss the new array - putting it on two lines means you can't miss it. (well, hopefully its not missed)